utils/pvacapi/app.py

```diff
@@ -27,13 +28,18 @@ def main():
     class IntConverter(BaseIntConverter):
         regex = r'-?\d+'
 
+    # determine IP address and setup CORS
+    IPAddr = None
+    if args.ip_address is None:
+        IPAddr = getIpAddress()
```

Comment:
pVACapi doesn't do a very good job following PEP8 conventions but variable names should be under_score, not camelCase.